example/src/main/java/org/hibernate/example/reactive/Main.java

```diff
@@ -46,7 +46,11 @@ public static void main(String[] args) {
 				//retrieve a Book
 				session -> session.find(Book.class, book1.id)
 						//print its title
-						.thenAccept( book -> out.println(book.title + " is a great book!") )
+						.thenCompose( book -> {
+							out.println(book.title + " is a great book!");
+							return session.fetch( book, Book_.isbn )
+									.thenAccept( isbn -> out.println(isbn + " is the ISBN") );
+						} )
```

Comment:
Folks, this is what it looks like to use this feature.

Anybody have any thoughts? Is this a good feature for us to have?

Reply:
LGTM